src/raftstore/store/fsm/apply.rs

```diff
@@ -485,8 +485,8 @@ pub fn notify_req_region_removed(region_id: u64, cb: Callback) {
 /// Calls the callback of `cmd` when it can not be processed further.
 fn notify_stale_command(tag: &str, term: u64, mut cmd: PendingCmd) {
     info!(
-        "{} command at [index: {}, term: {}] is stale, skip",
-        tag, cmd.index, cmd.term
+        "command is stable, skip";
+        "tag" => tag, "index" => cmd.index, "term" => cmd.term
```

Comment:
em, seem we should use region insted of tag here.

Reply:
I think region id is contained in `tag`.

Reply:
Oh, I know what you mean.